
// Сюда будут собираться полезные исключения, которые время от времени требуется кидать.

throw new RuntimeException; // в ходе работы программы можно кинуть исключение, например, при возникновении нежелательных условий

ArithmeticException ошибка при выполнении арифметического вычисления (например, с попыткой целочисленного деления на нуль). Класс ArithmeticalException унаследован от RuntimeException
